service/policy/objects.proto

```diff
@@ -262,6 +307,21 @@ message KeyAccessServer {
   common.Metadata metadata = 100;
 }
 
+message Key {
+  string id = 1;
+
+  bool is_active = 2;
```

Comment:
Might be better to use `google.protobuf.BoolValue`here so you can differentiate between set and false, set and true, and unset? https://protobuf.dev/reference/protobuf/google.protobuf/#bool-value This is particularly useful when marshaling/unmarshaling to JSON as an unset field value will become false (Go zero value) without this rather than skipped in the JSON.

Reply:
Same with `was_used`?